<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-27T14:42:01 IST" name="Default suite" finished-at="2023-11-27T14:42:20 IST" duration-ms="19584">
    <groups>
    </groups>
    <test started-at="2023-11-27T14:42:01 IST" name="Default test" finished-at="2023-11-27T14:42:20 IST" duration-ms="19584">
      <class name="Module1_Login_Test.SwagLoginTest">
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:Module1_Login_Test.SwagLoginTest@175c2241]" started-at="2023-11-27T14:42:01 IST" name="openBrowser" finished-at="2023-11-27T14:42:10 IST" duration-ms="9229" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method is-config="true" signature="loginToApp()[pri:0, instance:Module1_Login_Test.SwagLoginTest@175c2241]" started-at="2023-11-27T14:42:10 IST" name="loginToApp" finished-at="2023-11-27T14:42:15 IST" duration-ms="4506" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApp -->
        <test-method signature="VerifyTitle()[pri:0, instance:Module1_Login_Test.SwagLoginTest@175c2241]" started-at="2023-11-27T14:42:15 IST" name="VerifyTitle" finished-at="2023-11-27T14:42:16 IST" duration-ms="981" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failed: both results are failed expected [secret_sauce] but found [Swag Labs]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failed: both results are failed expected [secret_sauce] but found [Swag Labs]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at Module1_Login_Test.SwagLoginTest.VerifyTitle(SwagLoginTest.java:53)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyTitle -->
        <test-method is-config="true" signature="logoutFromApp(org.testng.ITestResult)[pri:0, instance:Module1_Login_Test.SwagLoginTest@175c2241]" started-at="2023-11-27T14:42:16 IST" name="logoutFromApp" finished-at="2023-11-27T14:42:20 IST" duration-ms="4697" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=VerifyTitle status=FAILURE method=SwagLoginTest.VerifyTitle()[pri:0, instance:Module1_Login_Test.SwagLoginTest@175c2241] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApp -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:Module1_Login_Test.SwagLoginTest@175c2241]" started-at="2023-11-27T14:42:20 IST" name="closeBrowser" finished-at="2023-11-27T14:42:20 IST" duration-ms="125" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- Module1_Login_Test.SwagLoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
